src/AM.Condo/Targets/Compile/Go.targets

```
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="GoBuild" Condition=" !$(CI) AND $(GoBuild) ">
+
+    <PropertyGroup>
+      <GoBuildOptions Condition=" '$(GoBuildOptions)' == '' ">$(GO_BUILD_OPTIONS)</GoBuildOptions>
+      <GoFileExtension Condition= " '$(IsWinodws)' == 'true' ">.exe</GoFileExtension>
```

```suggestion
      <GoFileExtension Condition= " $(IsWindows) ">.exe</GoFileExtension>
```

There was a typo and condo variables don't need to be single quoted as they will always be available (as long as the prepare target is called).

---

I gave that a try and condo errored out saying IsWindows is ''

that's why I left it like this, will take another look tho

---

you had a typo in the name; though, which might be why you got that error ;-)

---

`IsWinodws` vs `IsWindows`

---

WHAT?! how did that even work then :oh-me-gee:!

---

well; it would, except on windows :-D